contributed_definitions/NXxpcs.nxdl.xml

```diff
@@ -507,7 +507,7 @@
             roi index array or labeled array
 
             The values of this mask index (or map to) the :math:`Q` value from the
-            the ``dynamic_q_list`` field. Not that the value of ``0`` represents in-action. XPCS computations
+            ``dynamic_q_list`` field. Not that the value of ``0`` represents in-action. XPCS computations
```

Comment:
```suggestion
            ``dynamic_q_list`` field. Note that the value of ``0`` represents in-action. XPCS computations
```

Here, should "in-action" be "inaction"? Or does it mean something like "in motion"?  Would "uniform mode" or "unmodulated" be more appropriate?